### ISSUE TITLE
Missing items in JEI and related crashes

### ISSUE THREAD

**Issue by Nirvana0137** [FTBTeam/FTB-Presents-Direwolf20-1.12#15]
* **Modpack Version**: 2.0

* **Issue**: Several items like Thermal Foundation's Hardened Glass, several Immersive Engineering and Twilight Forest items are missing in JEI. There are just empty spaces in JEI and the items are not obtainable in survival mode. Further when opening a chest where one of the missing items like hardened glass are stored causes a crash of the game. I have added several mods (also Optifine) to the pack. However, I have tested two clean installs from the Twitch Client as well and had the same issue.

* **Link to Log or Crash File Paste**: https://paste.feed-the-beast.com/view/05af9866

* **Is it Repeatable?**: Yes it is repeatable with clean install of DW20 v2.0 from Twitch Client

* **Mod/s Affected**: Immersive Engineering, Thermal Foundation, Twilight Forest and maybe more

* **Known Fix**: No known Fix

**Comment by SnowShock35** [comment]
Thanks for the report. Please, could you go ahead and create a clean instance of the direwolf20 2.0.0 pack and run it without OptiFine. Once the pack is loaded try and create a new world and see if the items you described as missing now appear in JEI and whether or not hardened glass (which you can spawn in using the command /give) crashes your client when it tries to render it in your inventory.

If you are still having either of these issues let me know and once the new update is out we can try again to see if that fixes your issue. 

Thanks for helping to better the packs stability. 👍

**Comment by Nirvana0137** [comment]
I have installed 3 clean instances of the direwolf20 2.0.0 in a way that I downloaded the v 1.11.0 and updated it to 2.0.0 and in 2/3 cases the same error was occurring (missing items for IE, TF and TWF). However, the third instance is running fine without any missing items. I think that there is some problem occurring during the update to v 2.0.0. The third one also works fine with all my additional mods like e.g. Mekanism & Optifine installed.

Thank you very much for your effort and help
